includes/controllers/class-user.php

```diff
@@ -175,6 +175,14 @@ public function __construct( $args = [] ) {
 						'rest'   => true,
 					],
 
+					'user_resend_email_verify'     => [
+						'base'   => 'users_resource',
+						'path'   => '/resend-email-verify',
```

Comment:
It's better to follow existing URL patterns, please change this to the same structure as the password reset (I don't remember exactly, maybe /users/resend-email).